src/Sites/youtube.com/youtube.tsx

```diff
@@ -163,22 +163,26 @@ export class YouTubePageScript {
 	 */
 	private scrapeChannelID(): string {
 		const inputRenderer = (document.getElementsByTagName('yt-live-chat-renderer')[0] as HTMLElement & { __data: any; });
-		let user: YouTubePageScript.InternalUserData | null = null;
+		let channelID = '';
 		try {
-			user = inputRenderer.__data.data
+			channelID = inputRenderer
+				.__data
+				.data
 				.actionPanel
 				.liveChatMessageInputRenderer
 				.sendButton
 				.buttonRenderer
 				.serviceEndpoint
 				.sendLiveChatMessageEndpoint
-				.actions[0]
-				.addLiveChatTextMessageFromTemplateAction
-				.template
-				.liveChatTextMessageRenderer;
+				.params;
+
+			channelID = atob(decodeURIComponent(atob(channelID)))
+			.split(`*'\n\u0018`)[1]
+			.split('\u0012\u000b')[0];
+
```

Comment:
Is this removing the regular fallback? I believe this will break implementation in certain views, such as watching a VOD